graylog2-web-interface/src/components/nodes/NodesActions.jsx

```diff
@@ -38,6 +39,7 @@ const NodesActions = React.createClass({
     }
   },
   render() {
+    const apiBrowserURI = new URI(this.props.node.transport_address).directory('api-browser');
```

Comment:
I'm not 100% sure, but this could cause a problem if the transport address contains a path segment with trailing slash:
```
> URI("http://example.com/foo").directory("bar").toString()
"http://example.com/bar/foo"
> URI("http://example.com/foo/").directory("bar").toString()
"http://example.com/bar/"
```